sdk/identity/identity/review/identity.api.md

```diff
@@ -0,0 +1,125 @@
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+export { AccessToken }
+
+// @public
+export class AggregateAuthenticationError extends Error {
+    constructor(errors: any[]);
+    // (undocumented)
+    errors: any[];
+}
+
+// @public
+export const AggregateAuthenticationErrorName = "AggregateAuthenticationError";
+
+// @public
+export class AuthenticationError extends Error {
+    constructor(statusCode: number, errorBody: object | string | undefined | null);
+    // Warning: (ae-forgotten-export) The symbol "ErrorResponse" needs to be exported by the entry point index.d.ts
+    // 
+    // (undocumented)
+    readonly errorResponse: ErrorResponse;
+    // (undocumented)
+    readonly statusCode: number;
+}
+
+// @public
+export const AuthenticationErrorName = "AuthenticationError";
+
+// @public
+export class AuthorizationCodeCredential implements TokenCredential {
+    constructor(tenantId: string, clientId: string, clientSecret: string | undefined, authorizationCode: string, redirectUri: string, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public (undocumented)
+export type BrowserLoginStyle = "redirect" | "popup";
+
+// @public
+export class ChainedTokenCredential implements TokenCredential {
+    constructor(...sources: TokenCredential[]);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export class ClientCertificateCredential implements TokenCredential {
+    constructor(tenantId: string, clientId: string, certificatePath: string, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export class ClientSecretCredential implements TokenCredential {
+    constructor(tenantId: string, clientId: string, clientSecret: string, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export class DefaultAzureCredential extends ChainedTokenCredential {
+    constructor(identityClientOptions?: IdentityClientOptions);
+}
+
+// @public
+export class DeviceCodeCredential implements TokenCredential {
+    // Warning: (ae-forgotten-export) The symbol "DeviceCodePromptCallback" needs to be exported by the entry point index.d.ts
+    constructor(tenantId: string, clientId: string, userPromptCallback: DeviceCodePromptCallback, options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    }
+
+// @public
+export class EnvironmentCredential implements TokenCredential {
+    constructor(options?: IdentityClientOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+}
+
+// @public (undocumented)
```

Comment:
You've got a few undocumented warnings. Might as well fix those up while you're at it.

Reply:
Yep, a few unexported symbol warnings too, I'll get all of them.  Thanks!

Reply:
Done!